Add blog post for sprint 189 (#495)
openSUSE/obs-landing: _posts/development/2025-05-14-foster-collaboration.md

---
layout: post
title:  Introducing Assignments
category: development
---
Over the last few days, we’ve been focused on improving how responsibility is tracked in packages. Today, we’re excited to introduce Assignments.

These updates are part of the `Foster Collaboration` beta programs. You can find more information about the beta program [here](/2018/10/04/the-beta-program/).

{% include partials/_series-of-posts-about-foster-collaboration.md %}

### Using Assignments in the UI

Assignments allows collaborators to designate someone as the responsible person for a specific package for a specific amount of time.
The assignment makes it clear who’s currently responsible for that package. It helps prevent confusion, ensures clear ownership, and supports better coordination across the team.

> Assignments can be updated as responsibilities shift, keeping things flexible and transparent. An assignment expires after 24 hours.

Reviewer: ```suggestion
Assignments can be updated as responsibilities shift, keeping things flexible and transparent. Bear in mind that they expire after 24 hours.
```
Avoid repeating "assignment" too much.
